src/v3/test/integration/__snapshots__/admin-consent-without-logo.test.tsx.snap

```diff
@@ -70,11 +70,14 @@ exports[`admin-consent-without-logo should render form without logo 1`] = `
                 <div
                   class="MuiBox-root emotion-16"
                 >
+                  <div
+                    class="MuiBox-root emotion-17"
+                  />
```

Comment:
Hmm, it looks like an empty div is being added, curious where this is coming from?

Reply:
These were being added because all of the consent flows have `IDX_STEP.CONSENT_ADMIN` as their `nextStep` which is one of the conditions for hiding the `IdentifierContainer`, but I hadn't added that check before adding IdentifierContainers in the custom transformer when they're in the transaction context